valhalla/thor/bidirectional_astar.h

```diff
@@ -112,6 +112,9 @@ class BidirectionalAStar : public PathAlgorithm {
   std::vector<sif::BDEdgeLabel> edgelabels_reverse_;
   uint32_t max_reserved_labels_count_;
 
+  // if `true` clean reserved memory for edge labels
+  bool clear_reserved_memory_;
```

Comment:
all of these algorithms derive from pathalgorithm. rather than duplicating this maybe we should just move it up there and have less code in total? although i guess thats the same for all of the already duplicated stuff wrt edge labels handling.

Reply:
if it's possible we should move `max_reserved_labels_count_` also